seacrowd/sea_datasets/my_paraphrase/my_paraphrase.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{htay2022deep,
+  title={Deep Siamese Neural Network Vs Random Forest for Myanmar Language Paraphrase Classification},
+  author={Htay, Myint Myint and Thu, Ye Kyaw and Thant, Hnin Aye and Supnithi, Thepchai},
+  journal={Journal of Intelligent Informatics and Smart Technology},
+  year={2022}
+}
+"""
+
+_DATASETNAME = "my_paraphrase"
+
+_DESCRIPTION = """\
+The myParaphrase corpus is intended for the task of assessing whether pairs of Burmese sentences exhibit similar meanings \
+or are paraphrases. It encompasses 40461 pairs for training, along with 1000 pairs for an open test and an additional 1000 pairs \
+for a closed test. If a pair of sentences in Burmese is considered a paraphrase, it is labeled with "1"; if not, they receive a label of "0."
+"""
+
+_HOMEPAGE = "https://github.com/ye-kyaw-thu/myParaphrase"
+
+_LANGUAGES = ["mya"]
+
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: [
+        "https://github.com/ye-kyaw-thu/myParaphrase/raw/main/corpus/ver1.0/csv-qqp/train.csv",
+        "https://github.com/ye-kyaw-thu/myParaphrase/raw/main/corpus/ver1.0/csv-qqp/open-test.final.manual.csv",
+        "https://github.com/ye-kyaw-thu/myParaphrase/raw/main/corpus/ver1.0/csv-qqp/closed-test.csv",
+    ],
+}
+
+_SUPPORTED_TASKS = [Tasks.SEMANTIC_SIMILARITY]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+_TAGS = [0, 1]
```

Comment:
Mind trying to change this into list of string integers and see if that solves your issue? it's used for label naming on `utils/schemas/pairs.py`, and the corresponding value to be put in the example (`_generate_examples`) is [0,n-1] integers, which you already implemented correctly in L152.

```suggestion
_TAGS = ["0", "1"]
```